### DISCUSSION TITLE
Building Open-WATCOM on Linux hangs at step gendev mono01.cop (bld/browser/nt386) [CLOSED]

### DISCUSSION THREAD

**Discussion by ghost** [open-watcom/open-watcom-v2#848] [Q&A] [answered]
Hi,

I'm trying to build Open-WATCOM on a Linux host. Need the  tools for another project. But I run into an issue with the uttility `gendev`.

It hangs here:
```
================== 05:53:07 /home/build/ow/bld/browser/nt386 ==================
%make prepare_gendev_tmp_batch
gendev mono01.cop
ERROR: unknown window sizes li=0  co=0, setting to 80x25
```
No response from the build system.

Have also tried to use gendev manually for the specific file (mono01.cop). But get this output from gendev:
![gendev](https://user-images.githubusercontent.com/20015689/160536158-ab06ae5d-64d9-40c8-ad4f-b80f5506cc2c.png)
I'm using FreeDOS 1.3. Also not sure, if this is an issue from DOSEMU.

Can somebody help me?

**UPDATE:**

Okay, it was an usage issue from me, with the utility `gendev`. `mono01.cop` is the output file...
![mono01 fon](https://user-images.githubusercontent.com/20015689/160538385-06a5952f-b523-4ed0-8fe5-cee50c7da6c9.png)
Must investigate the Open-WATCOM Makefile...

**Comment by jmalak** [comment]
It looks like the issue with DOSEMU.
Try to configure OW build [to use DOSBOX](https://github.com/open-watcom/open-watcom-v2/wiki/Build#troubleshooting) instead of DOSEMU.

**Comment by ghost** [comment, marked as answer]
I have solved it by myself in the past...

But I don't provide the information how...

Closed.
